scripts/index.js

```diff
@@ -0,0 +1,182 @@
+const mainSkills = document.getElementById("main-skills");
+const mainPortfolio = document.getElementById("main-portfolio");
+const mainVideo = document.getElementById("main-video");
+const mainPrice = document.getElementById("main-price");
+const mainFeedback = document.getElementById("main-feedback");
+const headerItemsRightPartList = document.getElementById("header-items-right-part-list");
+headerItemsRightPartList.addEventListener('click', (event) => {
```

Comment:
We can do this logic event without JS. We can add links instead of buttons and make some refs for them in order to move into target place on the page.